### ISSUE TITLE
[tests] Codecoverage and analyzer tests verification

### ISSUE THREAD

**Issue by Eshaan7** [intelowlproject/IntelOwl#558]
Few problems are:
- [x]  Since the analyzer scripts test are dynamic, we need a way to verify/check that each and every analyzer from `analyzer_config.json` was ran. 
     - [x] For the observable analyzers, the `analyzers_to_execute` is calculated dynamically. So here we need a check.
     - [x] For the file analyzers, the `analyzers_to_execute` is defined as static lists. So here only one time manual checking is needed.
- [x]  We are executing tests from the `uwsgi` container but the analyzers are run in the `celery` container. Because of this coverage for the analyzers won't be reported. This can be fixed either by combining coverages from all containers (not sure how to do this) or using the celery's `task_always_eager` option so the analyzers are executed as part of the `uwsgi` container only. (PR: #599, #606)
- [x] The django test suite by default uses a different `_test` suffixed DB. But I could not get celery to work with this so I had to overwrite this configuration so the tests run in the main DB (the `DB_NAME` env variable) only. This may also be fixed by the celery's `task_always_eager` option. (PR: #607)

**Comment by Eshaan7** [comment]
The following analyzers are missing from `test_file_scripts`. (I forgot `run_hash` ones)

```
['Cymru_Hash_Registry_Get_File',
 'HybridAnalysis_Get_File',
 'MISPFIRST_Check_Hash',
 'MISP_Check_Hash',
 'MalwareBazaar_Get_File',
 'OTX_Check_Hash',
]
```

**Comment by Eshaan7** [comment]
Since the main issue is coverage reporting, we will tackle this when we have to merge into develop/master.

**Comment by Eshaan7** [comment]
Codecov reporting was broken. Was fixed in #599.
